pyiron_base/job/worker.py

```diff
@@ -126,7 +126,7 @@ class WorkerJob(PythonTemplateJob):
 
     def __init__(self, project, job_name):
         super(WorkerJob, self).__init__(project, job_name)
-        self.input.project = None
+        self.input.project = project.path
```

Comment:
```suggestion
        if not state.database.database_is_disabled:
            self.input.project = project.path
        else: 
            self.input.project = self.working_directory
```
I just noticed that we also have to consider the case without database.